client-lite/src/exe/docs.cpp

```diff
@@ -119,26 +120,11 @@ class BoostAsioService
     std::thread _workerThread;
 };
 
-class CurlGlobalInitCall
-{
-public:
-    CurlGlobalInitCall()
-    {
-        auto result = curl_global_init(CURL_GLOBAL_ALL);
-        THROW_HR_IF(E_OUTOFMEMORY, result != CURLE_OK);
-    }
-
-    ~CurlGlobalInitCall()
-    {
-        curl_global_cleanup();
-    }
-};
-
 HRESULT Run() try
 {
     InitializeDOPaths();
 
-    CurlGlobalInitCall curlGlobalInit;
+    CurlGlobalInit curlGlobalInit;
```

Comment:
Can't CurlGlobalInit be initialized in the DownloadManager constructor?

Reply:
Since it is a global setup, it is safer to do it here rather than in DownloadManager where it would have to be done before CurlRequests is constructed. Easy to get it wrong there.